corehq/apps/export/models/new.py

```diff
@@ -2735,26 +2735,38 @@ def get_headers(self, split_column=False):
         return [header_template.format(header) for header_template in header_templates]
 
     def get_value(self, domain, doc_id, doc, base_path, split_column=False, **kwargs):
-        value = super(SplitGPSExportColumn, self).get_value(
+        coord_string = super().get_value(
             domain,
             doc_id,
             doc,
             base_path,
             **kwargs
         )
         if not split_column:
-            return value
+            return coord_string
 
-        if value == MISSING_VALUE:
-            return [MISSING_VALUE] * 4
+        return self.extract_coordinate_array(coord_string)
 
-        values = [EMPTY_VALUE] * 4
+    @classmethod
+    def extract_coordinate_array(cls, coord_string):
```

Comment:
It looks like `cls` is unused. You could go for
```suggestion
    @staticmethod
    def extract_coordinate_array(coord_string):
```